core23/shariff-bundle/3.0/config/packages/core23_shariff.yaml

```diff
@@ -0,0 +1,15 @@
+core23_shariff:
+    services:
+        facebook:
+            app_id:  '%env(FACEBOOK_API_ID)%'
+            secret:  '%env(FACEBOOK_API_SECRET)%'
+
+sonata_block:
+    blocks:
+        core23_shariff.block.share:
+
+doctrine_cache:
+    providers:
+        core23_shariff:
```

Comment:
It looks a bit weird that you've hard coded dependency to `doctrine_cache.providers.core23_shariff`. You could let the user provide an id to a cache service. 

(Also consider using PSR6)